examples/cookbook/lists/floating_app_bar/lib/main_cupertino.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/cupertino.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    const title = 'Floating Navigation Bar';
+
+    return CupertinoApp(
+      title: title,
+      home: CupertinoPageScaffold(
+        // No navigation bar provided to CupertinoPageScaffold,
+        // only a body with a CustomScrollView.
+        child: CustomScrollView(
+          slivers: [
+            // Add the navigation bar to the CustomScrollView.
+            const CupertinoSliverNavigationBar(
+              // Provide a standard title.
+              largeTitle: Text(title),
+            ),
+            // #docregion SliverList
+            // Next, create a SliverList
+            SliverList(
+              // Use a delegate to build items as they're scrolled on screen.
+              delegate: SliverChildBuilderDelegate(
+                // The builder function returns a ListTile with a title that
+                // displays the index of the current item.
+                (context, index) =>
+                    CupertinoListTile(title: Text('Item #$index')),
+                // Builds 50 ListTiles
+                childCount: 50,
+              ),
+            ),
```

Comment:
Sorry, this code I gave you isn't ideal. While this is correct, we can use higher-level APIs that is a little less verbose:

```suggestion
            SliverList.builder(
              // The builder function returns a ListTile with a title that
              // displays the index of the current item.
              itemBuilder: (context, index) =>
                  CupertinoListTile(title: Text('Item #$index')),
              // Builds 50 ListTiles
              itemCount: 50,
            ),
```

It looks like the Material example can be updated too.

Reply:
If you use the higher-level API, don't forget to update this sentence as well: https://github.com/flutter/website/pull/11821/files#diff-5558f4bc5553a504bbe7bf7926f93f9c0d42440c9a0de3fd0df29a5c8504ac69R188-R190

Reply:
No worries at all. I've updated the code for Material and Cupertino. Can you double-check my changes?